go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/kr/pretty v0.2.0
 	github.com/mattn/go-shellwords v1.0.5
 	github.com/miekg/dns v1.0.14
-	github.com/mitchellh/protoc-gen-go-json v0.0.0-20200113165135-fd297ce346f1
+	github.com/mitchellh/protoc-gen-go-json v0.0.0-20200717004536-8fbb6f3d7291
```

Comment:
just a note, this tool is still using deprecated `github.com/golang/protobuf` package as well.

Reply:
It's not the whole of `github.com/golang/protobuf` that is deprecated but just `github.com/golang/protobuf/protoc-gen-go`.

And apparently only `github.com/envoyproxy/protoc-gen-validate` still imports it:
```
$ go mod why "github.com/golang/protobuf/protoc-gen-go"
# github.com/golang/protobuf/protoc-gen-go
github.com/cilium/cilium/pkg/hubble/tools
github.com/envoyproxy/protoc-gen-validate
github.com/golang/protobuf/protoc-gen-go
```

Reply:
You are right, thanks :100: 

Actually, I meant this deprecated package `github.com/golang/protobuf/jsonpb` which is used in protoc-gen-go-json.

Reply:
Yes, there's an issue for this but it's still open: https://github.com/mitchellh/protoc-gen-go-json/issues/6